src/components/profile/Profile.js

```diff
@@ -141,6 +141,7 @@ export function Profile({ match }) {
                 await dispatch(loadRecoveryMethods())
                 dispatch(getAccessKeys(accountId))
                 dispatch(getLedgerKey())
+                await dispatch(getBalance())
```

Comment:
@marcinbodnar [not a blocker for this PR]: seems like we can improve even further if parallelise kicking off all the queries vs using `await` to do some of them sequentially? Basically kick off loading balance when we know that we need to do it vs when recovery methods already loaded, etc

Reply:
@vgrichina Right, in this particular case I wanted to load Recovery Methods before fetching for staking balance details, which might take few seconds.